doc/changelogs/CHANGELOG_V14.md

```diff
@@ -31,6 +32,145 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="14.2.0"></a>
+## 2020-05-05, Version 14.2.0 (Current), @targos
+
+### Notable Changes
+
+#### Track function calls with `assert.CallTracker` (experimental)
+
+`assert.CallTracker` is a new experimental API that allows to track and later
+verify the number of times a function was called. This works by creating a
+`CallTracker` object and using its `calls` method to create wrapper functions
+that will count each time they are called. Then the `verify` method can be used
+to assert that the expected number of calls happened:
+
+```js
+const assert = require('assert');
+
+const tracker = new assert.CallTracker();
+
+function func() {}
+// callsfunc() must be called exactly twice before tracker.verify().
+const callsfunc = tracker.calls(func, 2);
+callsfunc();
+callsfunc();
+
+function otherFunc() {}
+// The second parameter defaults to `1`.
+const callsotherFunc = tracker.calls(otherFunc);
+callsotherFunc();
+
+// Calls tracker.verify() and verifies if all tracker.calls() functions have
+// been called the right number of times.
+process.on('exit', () => {
+  tracker.verify();
+});
+```
+
+Additionally, `tracker.report()` will return an array which contains information
+about the errors, if there are any:
+
+```js
+const assert = require('assert');
+
+const tracker = new assert.CallTracker();
+
+function func() {}
+const callsfunc = tracker.calls(func);
+
+console.log(tracker.report());
+/*
+[
+  {
+    message: 'Expected the func function to be executed 1 time(s) but was executed 0 time(s).',
```

Comment:
How can I avoid the linter error here (too long line) ?
I don't want to add an eslint-disable comment in an example

Reply:
Thanks, fixed.